manipulation/src/onnx.pb.cc

```diff
@@ -0,0 +1,8946 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Having yet another checked-in copy of protobuf is messy. Can we depend on generated protoc files from onnx binary installation or just invoke protoc as a part of cmake script directly?

Reply:
probably - this was the quickest way to get things working, but I'll try one of those approaches